source/clear-linux/tutorials/wp-install/wp-install.rst

```diff
@@ -3,204 +3,150 @@
+This tutorial follows these steps:
 
-We are setting up a WordPress blog which can be accessed at:
+1. Download WordPress and add components to the necessary directory. 
+2. Set up WordPress by entering information in the WordPress admin console.
+3. Run the installer to fully install WordPress on your system and create a username and password for your newly created blog. 
+
+Note that throughout this tutorial, we reference your website name as <your_website>. 
 
-http://<your_website>/blog
 
-To accomplish this setup, we must put the WordPress components in the
-:file:`/var/www/html/blog` directory.
+Install WordPress
+--------------------
+
+For this tutorial, we will create a WordPress blog that can be accessed at:
+
+http://<your_website>/blog
 
-Let us get started:
+To accomplish this setup, you must add WordPress components to the :file:`/var/www/html/blog` directory.
 
+Follow these steps:
 
-1. To move to the top-level of the website’s root directory, enter the
-   following command:
+1. To move to the top level of the website’s root directory, enter thefollowing command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       cd /var/www/html
 
 2. To download the latest version of WordPress, enter the following command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo curl -O https://wordpress.org/latest.tar.gz
 
-3. To extract all the files and directories from the downloaded file,
-   enter the following command:
+3. To extract all files and directories from the downloaded file, enter the following command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo tar –xzvf latest.tar.gz
 
-4. To rename the top-level WordPress directory to “blog”, enter the following
-   command:
+4. To rename the top-level WordPress directory to “blog,” enter the following command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo mv wordpress blog
 
 5. To remove the downloaded tar file, enter the following command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo rm latest.tar.gz
 
-Setting up WordPress
+Set up WordPress
 --------------------
 
-With the WordPress components loaded into the
-:file:`/var/www/html/blog` directory, we can set everything up.
+Recall that you created a database and user when you installed phpMyAdmin during :ref:`Creating a Clear Linux based web server <web-server-install>`. To continue with the WordPress setup, go to: http://<your_website>/blog/wp-admin/install.php.
 
-Instead of editing the :file:`wp_config.php` file manually, we are using the
-web-based configuration tool to setup the database name and user. We created
-both in the :ref:`web server tutorial <web-server-install>`. In your browser,
-go to: http://<your_website>/blog/wp-admin/install.php.
-
-Your screen should look like figure 1:
+The WordPress language option screen appears, as shown in figure 1. Select :guilabel:`English` and click :guilabel:`Continue`.
```

Comment:
(Use procedural steps now) 
Revise to: 
#.  Select :guilabel:`English` and click :guilabel:`Continue`.

Reply:
done